src/libraries/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -346,7 +346,10 @@ public static int WindowWidth
                 GetWindowSize(out int width, out _);
                 return width;
             }
-            set { throw new PlatformNotSupportedException(); }
+            set
+            {
+                 SetWindowSize(value, WindowHeight);
+            }
```

Comment:
nit: you could use expression here
```suggestion
            set => SetWindowSize(value, WindowHeight);
```